content/doc/developer/initialization/index.adoc

```diff
@@ -0,0 +1,9 @@
+---
+title: Initialization and Startup
+layout: developerchapter
+wip: true
+---
+
+////
+https://wiki.jenkins-ci.org/display/JENKINS/Jenkins+Pieces+in+GitHub
```

Comment:
Right, left it out because of the terrible URL, needs a different label than page title.